t/tests.t

```diff
@@ -1488,7 +1488,7 @@ my @list = (
     '"Greg Norris (humble visionary genius)" <nextrightmove-- ATAT --bang.example.net>, <advocacy-- ATAT --p.example.org>',
     [
       [
-        'Greg Norris',
+        'Greg Norris ',
```

Comment:
This is incorrect change. String `"Greg Norris (humble visionary genius)" <nextrightmove-- ATAT --bang.example.net>` (after replacing `-- ATAT --` with `@`) must be parsed as:
* phrase: `Greg Norris (humble visionary genius)`
* mailbox: `nextrightmove`
* domain: `bang.example.net`

You are missing part after space in parenthesis in phrase part. In quoted string content of parenthesis is not comment which you probably missed.

Reply:
Ah, this is fixed in next commit. In this commit is just test noise.

Reply:
Yes, I tried to explain that in the commit message, sorry if it wasn't clear enough.